test/integration/services/data/test-update-reduction.js

```diff
@@ -49,6 +52,16 @@ describe('/services/data/update-reduction', function () {
       })
   })
 
+  it('should update a reduction status and return an id ', function () {
```

Comment:
Should this be in it's own separate file, seeing as its testing a new file

Reply:
Yes, it should.